main/registerIpcMainActionListeners.ts

```diff
@@ -57,7 +57,32 @@ export default function registerIpcMainActionListeners(main: Main) {
       const backupPath = path.join(dbsPath, 'backups');
       await fs.ensureDir(backupPath);
 
-      return path.join(dbsPath, `${companyName}.books.db`);
+      let dbFilePath = path.join(dbsPath, `${companyName}.books.db`);
+
+      if (await fs.pathExists(dbFilePath)) {
+        const option = await dialog.showMessageBox({
+          type: 'question',
+          title: 'File Exists',
+          message: `Filename already exists. Do you want to overwrite the existing file or create a new one?`,
+          buttons: ['Overwrite', 'New'],
+        });
```

Comment:
Is it possible to display this message using `showDialog` Component ? 

Ref: [DatabaseSelector.vue](https://github.com/frappe/books/blob/394accfa4164a8928f3dea40bc6db59440f003ae/src/pages/DatabaseSelector.vue#L378-L390)